x-pack/plugins/serverless_search/public/application/components/connectors_ingestion.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiSpacer,
+  EuiText,
+  EuiLink,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import React from 'react';
+import { useCreateConnector } from '../hooks/api/use_create_connector';
+
+export const ConnectorIngestionPanel = () => {
+  const { createConnector } = useCreateConnector();
+  return (
+    <EuiFlexGroup direction="column" justifyContent="spaceEvenly" gutterSize="s">
+      <EuiFlexItem grow={false}>
+        <EuiTitle size="xxs">
+          <h6>
+            {i18n.translate(
+              'xpack.serverlessSearch.ingestData.alternativeOptions.connectorsTitle',
+              {
+                defaultMessage: 'Connectors',
+              }
+            )}
+          </h6>
+        </EuiTitle>
+        <EuiSpacer size="xs" />
+        <EuiText>
+          <p>
+            {i18n.translate(
+              'xpack.serverlessSearch.ingestData.alternativeOptions.connectorsDescription',
+              {
+                defaultMessage:
+                  'Sync third-party data sources to Elasticsearch, by deploying open code Elastic connectors on your own infrastructure.  ',
+              }
+            )}
+          </p>
+        </EuiText>
+      </EuiFlexItem>
+      <EuiFlexGroup direction="row" justifyContent="flexStart" alignItems="center">
+        <EuiFlexItem grow={false}>
+          <EuiLink onClick={() => createConnector()}>
+            {i18n.translate(
+              'xpack.serverlessSearch.ingestData.alternativeOptions.setupConnectorLabel',
+              {
+                defaultMessage: 'Set up a connector',
+              }
+            )}
+          </EuiLink>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiLink href="https://github.com/elastic/connectors-python">
+            <EuiButtonEmpty color="primary" iconType="logoGithub" size="s">
```

Comment:
`EuiButtonEmpty` has a `href` prop, so I don't think you need to nest `EuiButtonEmpty` inside `EuiLink` to use an `href`

Reply:
[design](https://www.figma.com/file/aRW3wtDRGgwOunC8vXIvdm/Milestone-0-UX?type=design&node-id=3893-443443&mode=design&t=fm8fIxX5ATxfT15B-0) has `external` for the first link, which I think isn't available in `EuiButtonEmpty`.

Reply:
Looking at some other code, I think you can just use [GithubLink](https://github.com/elastic/kibana/blob/main/packages/kbn-search-api-panels/components/github_link.tsx) component for this.

Reply:
We could, but i think [GithubLink](https://github.com/elastic/kibana/blob/main/packages/kbn-search-api-panels/components/github_link.tsx) has `external` to set to `true` by default.  To use, we will need to update [GithubLink](https://github.com/elastic/kibana/blob/main/packages/kbn-search-api-panels/components/github_link.tsx) to optionally set `external` and update all instance where we use this.

Reply:
GithubLink has external set to true because a Github link is always an external link, so it should always be external. If design wants us to link to Github, it needs to be an external link. If they have a github link that's not marked as external, that's a mistake on their part :)

Reply:
I suppose the  same logic  apply to `Docker` as well. As it take to GitHub docker documentation:
https://github.com/elastic/connectors/blob/main/docs/DOCKER.md

Reply:
All the component with GitHub logo use [GithubLink](https://github.com/elastic/kibana/blob/main/packages/kbn-search-api-panels/components/github_link.tsx) and modified [connectors_ingestion](https://github.com/elastic/kibana/blob/51da5af83162d9250be4abcbf53cdfb35a40a81f/x-pack/plugins/serverless_search/public/application/components/connectors_ingestion.tsx) , docker a bit to align with other GitHub links.
  
With this change UI:
![New UI](https://github.com/elastic/kibana/assets/55930906/c9e872bd-ba0a-465b-a1f9-3bf811b02465)

commit added in 1b2d6f14f1ed2b4affb65cd5ecbb8bf4fa086197